src/styles/antd-overrides/input.scss

```diff
@@ -2,6 +2,7 @@
 .ant-input-affix-wrapper .ant-input,
 .ant-input-textarea textarea {
   @apply text-primary;
+  @apply py-1.5;
```

Comment:
does this affect any other antd text areas on the site?

Reply:
Yeah, they've all haven't got enough top padding

Reply:
@wraeth-eth It's only a 2px change, just keeping inline with Strath's latest.

Before:
<img width="516" alt="Screen Shot 2023-08-22 at 12 31 33 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/9d6c74cd-aae8-4ecf-bb15-94d4aee0547e">

After:
<img width="513" alt="Screen Shot 2023-08-22 at 12 30 36 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/0c7be985-2f8d-4c1a-8d6a-40c5e68271d6">